JavaScriptLearnings.html

```diff
@@ -8,35 +8,41 @@
 		
 		var names = ['Max', 'Alex', 'Rach', 'John', 'Vlad','Richard', 'Natalie', 'Craig', 'Mike', 'Brenden'];
 		
-		for (item in names) {
-			//display = names[item];	
-    		// console.log(display);
-			
-			document.write(names[item]);
+		for (index in names) { //mweber: i would change item to index to make it clearer for other developers
+						
+			document.write(names[index]);
 			document.write('<br>');
-		}
 
-		document.write('<br>');
+			}
 
-		function dosomething (name){
-			/*console.log('Hello');
-			console.log(name);
-			console.log('do something');*/
-
-			return name;
-		}
+		document.write('<br>');
+		
+		/**
+         * TODO: use a better name for your function, what do you want to achieve with it
+         * If you do not find a good name it is probably not a good function
+         *
+         * TODO: Improve the method so that you call it down in the for loop and nothing else.
+         *
+         * This method accepts a name as input (which is most likly Alex, Richard...)
+         * And you return the method with the same name (which is literally the same)
+         *
+         * Also recommended is camelCase for function names :slightly_smiling_face: -> doSomething(name)
+         */
 
-		for (x in names) {
-			//displaying = names[x];
-			//console.log(dosomething(display));
-			
+		function assignTaskToNames (name){
 			document.write('Hello');
 			document.write('<br>');
-			document.write(dosomething(names[x]));
+			document.write(name);
 			document.write('<br>');
 			document.write('do something');
 			document.write('<br>');
 			document.write('<br>');
+			//return name;
+			}
+
+		for (eachPerson in names) {
```

Comment:
did you know that eachPerson is again an index like above.
```
for (index in names)
```
Do you know a better solution for this loop?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
I thought I've to use different variables, every time i use.
Will change in the new pull request